lib/stripe.js

```diff
@@ -1,147 +1,104 @@
-'use strict';
-
+"use strict";
+const _Error = require("./Error");
 const resources = require('./resources');
-
 const DEFAULT_HOST = 'api.stripe.com';
 const DEFAULT_PORT = '443';
 const DEFAULT_BASE_PATH = '/v1/';
 const DEFAULT_API_VERSION = null;
-
 const DEFAULT_TIMEOUT = 80000;
-
 Stripe.PACKAGE_VERSION = require('../package.json').version;
-
 const utils = require('./utils');
-const {determineProcessUserAgentProperties, emitWarning} = utils;
-
-Stripe.USER_AGENT = {
-  bindings_version: Stripe.PACKAGE_VERSION,
-  lang: 'node',
-  publisher: 'stripe',
-  uname: null,
-  typescript: false,
-  ...determineProcessUserAgentProperties(),
-};
-
+const { determineProcessUserAgentProperties, emitWarning } = utils;
+Stripe.USER_AGENT = Object.assign({ bindings_version: Stripe.PACKAGE_VERSION, lang: 'node', publisher: 'stripe', uname: null, typescript: false }, determineProcessUserAgentProperties());
```

Comment:
Why this uses `Object.assign` instead of a regular assignment?

Reply:
(also discussed over Slack) It looks like Object spread was included starting in ES2018. The `Stripe.USER_AGENT = Object.assign...` syntax gets changed back to the original `Stripe.USER_AGENT = {...}` if we update the target to ES2018, but it wouldn't be safe to bump the spec to ES2018 since older versions of Node that we support would error: https://node.green/#ES2018